src/Mono.Android/Mono.Android.targets

```diff
@@ -42,6 +42,17 @@
     />
     <Touch Files="Android.Runtime\JNIEnv.g.cs" />
   </Target>
+  <Target Name="_BuildJniMarshalMethodsGen"
+      BeforeTargets="CoreCompile"
+      Inputs="$(JavaInteropFullPath)\tools\jnimarshalmethod-gen\Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj"
+      Outputs="$(XAInstallPrefix)xbuild\Xamarin\Android\jnimarshalmethod-gen.exe;$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Runtime.Environment.dll.config">
+    <MSBuild
```

Comment:
Why is this needed?

Explicit `<MSBuild/>` invocations annoy Atsushi whenever files get out of sync. What *will* happen is that the `.csproj` isn't updated, but one of the source files for that project *is* updated, but the project won't necessarily be rebuilt, which later results in a "mismatch" that can only be fixed by `git clean -xdf`, and everyone is sad.

Additionally, I don't see *why* this is needed in the first place. `Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj` is added to `Xamarin.Android.sln`; therefore, it *should be built*, eventually. The `@(ProjectReference)` should also ensure that the project is built.

Why is this `<MSBuild/>` needed at all?